sdks/python/src/opik/llm_usage/opik_usage_factory.py

```diff
@@ -1,47 +1,53 @@
 import logging
-from . import opik_usage
-from typing import Dict, Any, Callable, Optional, Union
+from typing import Any, Callable, Dict, List, Optional, Union
+
 from opik.types import LLMProvider
+from . import opik_usage
 
 
-# TODO: it was discovered that one provider may have more than one token usage formats, and
-# sometimes we support one usage format but not the cost tracking for its provider.
-# So, this dictionary and corresponding logic should be updated.
-# We need to have one dict - USAGE_FORMAT: OPIK_USAGE_BUILDER, and probably
-# another dict - PROVIDER: USAGE_FORMATS.
-# See the ticket: OPIK-1407
 _PROVIDER_TO_OPIK_USAGE_BUILDER: Dict[
-    Union[str, LLMProvider], Callable[[Dict[str, Any]], opik_usage.OpikUsage]
+    Union[str, LLMProvider],
+    List[Callable[[Dict[str, Any]], opik_usage.OpikUsage]],
 ] = {
-    LLMProvider.OPENAI: opik_usage.OpikUsage.from_openai_completions_dict,
-    LLMProvider.GOOGLE_VERTEXAI: opik_usage.OpikUsage.from_google_dict,
-    LLMProvider.GOOGLE_AI: opik_usage.OpikUsage.from_google_dict,
-    LLMProvider.ANTHROPIC: opik_usage.OpikUsage.from_anthropic_dict,
-    "_bedrock": opik_usage.OpikUsage.from_bedrock_dict,
-    "_openai_responses": opik_usage.OpikUsage.from_openai_responses_dict,
+    LLMProvider.OPENAI: [
+        opik_usage.OpikUsage.from_openai_completions_dict,
+        opik_usage.OpikUsage.from_openai_responses_dict,
+    ],
+    LLMProvider.GOOGLE_VERTEXAI: [opik_usage.OpikUsage.from_google_dict],
+    LLMProvider.GOOGLE_AI: [opik_usage.OpikUsage.from_google_dict],
+    LLMProvider.ANTHROPIC: [opik_usage.OpikUsage.from_anthropic_dict],
+    "_bedrock": [opik_usage.OpikUsage.from_bedrock_dict],
```

Comment:
Let's add a comment that will describe why there is _bedrock so it's easier to understand when someone sees it for the first time